scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -534,22 +534,22 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             var deregisterTansa = $rootScope.$on('tansa:end', afterTansa);
 
             $scope.runTansa = function() {
-                const editor = editorResolver.get();
+                if (window.tansa && window.RunTansaProofing && window.tansaIsLoaded) {
```

Comment:
Will `if (window.tansaIsLoaded) {` not be enough?

Reply:
I think it is safer, for example RunTansaProofing is an object created by tansa script.